fastlane_bot/events/pools/solidly_v2.py

```diff
@@ -85,15 +85,15 @@ async def update_from_contract(
         """
         See base class.
         """
-        reserve_balance = await contract.caller.getReserves()
+        reserve_balance = contract.caller.getReserves()
 
         try:
-            factory_address = await contract.caller.factory()
+            factory_address = contract.caller.factory()
         except Exception:
             # Velocimeter does not expose factory function - call voter to get an address that is the same for all Velcoimeter pools
-            factory_address = await contract.caller.voter()
+            factory_address = contract.caller.voter()
 
-        self.is_stable = await contract.caller.stable()
+        self.is_stable = contract.caller.stable()
```

Comment:
How are all of these changes related to the issue that this PR is attempting to address?